Rovers/Rovers.tex

```diff
@@ -0,0 +1,344 @@
+These are skills that benefit RIT SPEX. 
+
+\section{Implementation}
+\label{sec:implementation}
+This project would involve divvying out the different project areas to the different team members or small groups and each week meeting to discuss progress. 
+The team would take this information, refine it and put it into the technical report along with construction.  
+
+\subsection{Deliverables}
+\label{subsec:deliverables}
+The deliverable includes the technical report, Rover software, and Rover hardware.
+The primary deliverables of this project will be the technical report and the rover itself by the end of the second semester.
+It is also noted that the materials that the team will come across or notes should also be saved in an Google Drive folder for future reference. 
+The team will also present at SPEX design reviews and the weekly checkups at general meetings with the areas the team members are working on that week.
+The project is intended to last 2 semesters with the deliverable of a basic rover and a technical report.   
+
+\subsubsection{Technical Report}
+\label{techreport}
+The technical report of the rover is a design document complete with the hardware, software, and electrical components of the rover. This report is important as it allows for a future team to easily learn from the past team's experience. This Technical Report will be hosted on GitHub. This will also include the results of the rover tests and the teams recommendations  based on them. 
+
+\subsubsection{Rover Software}
+\label{roversoftware}
+The rover software has to be written and hosted on GitHub. This will include rover features such as navigation, control, and power regulation. The rover need to be controlled so software must be made to do so. This means the rover electronics must be integrated with a control application. The rover will need to distribute power to the motors in the wheels so software will need to be made to regulate such power. There will also need to be software for the autonomous or partially autonomous features on the rover. 
+
+\subsubsection{Rover Hardware}
+\label{roverhardware}
+The rover will need to be constructed and tested. Because of this a physically constructed rover is needed. This will be necessary for ImagineRIT for cool factor. This includes the rover body, mast, suspension, and wheels. 
+
+\begin{figure}[ht!]
+  \includegraphics[width=\linewidth]{figs/rocker-bogie.png}
+  \caption{Rocker-Bogie mechanism}
+\label{fig:lifecycle}
+\end{figure}
+
+\subsection{Milestones}
+\label{subsec:milestones}
+The largest milestones for the technical report would be the would be draft and completion. 
+The rover itself would include the various CAD models of the rover, software milestones, electrical system completion, and drive train system. 
+Other milestones include securing funding and completion of the learning period at the beginning of the fall semester.
+
+\begin{table}[ht!]
```

Comment:
Will the MVP and all improvements follow the same timeline? Will some improvements take longer/require more people-hours than others?

Reply:
The improvements would be developed in tandem but would likely start later as they would be dependent on funding. The improvements require a variety of skills neccicary and because of this they would take different amounts of time.